tests/main/lxd-mount-units/task.yaml

```diff
@@ -41,7 +41,7 @@ execute: |
         lxd.lxc exec ubuntu -- sh -c "echo https_proxy=$https_proxy >> /etc/environment"
     fi
     # wait for the container to be fully up
-    (lxd.lxc exec ubuntu -- systemctl --wait is-system-running || true) | MATCH "(running|degraded)"
+    retry --wait 1 -n 10 sh -c "lxd.lxc exec ubuntu -- systemctl --wait is-system-running | grep -Eq (running|degraded)"
```

Comment:
This seems unrelated, I'd suggest moving into its own PR.

Reply:
I added this line because I reproduced the same issue in arm as it is happening with the kenetic images.